### ISSUE TITLE
002

### ISSUE THREAD

**Issue by suredream** [suredream/hack-your-ds-interview#2]
# Intro
- date: 11/20/2021
- host: @stanghong
- facilitator:  TBD
- topic:
- review page:

# Reference

**Comment by stanghong** [comment]
can you point me to your this week notebook which you will present? So I can follow the same format. Thanks

    On Thursday, November 4, 2021, 06:33:02 PM PDT, Jun Xiong ***@***.***> wrote:  
 
 


Assigned #2 to @stanghong.

**Comment by suredream** [comment]
This week (11/06/2021) we don't have an agenda. The 1st will be 13th this month.

This issue is created for you on 11/20/2021.

I have a full time job with three kids. So generally it is hard for me to fill the host role.  I would rather serve as an evangelist for this "ds-review" community. Again, tks for your proactive contribution @stanghong.

**Comment by stanghong** [comment]
Sure. let's make it sustainable, low-maintainance platform.

I will start my new role end of Nov... I do not expect too much work for organizing the talk.I do see the forum as a venue for future DS community to learn and improve.

    On Friday, November 5, 2021, 02:07:39 PM PDT, Jun Xiong ***@***.***> wrote:  
 
 


This week (11/06/2021) we don't have an agenda. The 1st will be 13th this month.

This issue is created for you on 11/20/2021.

I have a full time job with three kids. So generally it is hard for me to fill the host role. I would rather serve as an evangelist for this "ds-review" community. Again, tks for your proactive contribution @stanghong.

**Comment by suredream** [comment]
Great. Did you reply github issues in email client instead of github.com? If so, you can stop quote the previous email in your reply. So we can keep the issue area neat and easy to browse.

**Comment by suredream** [comment]
Everything looks good now. Let's try to get more people involved with the review.

**Comment by suredream** [comment]
- intro
     a) 由 host 简要说明其原理、应用前提、优缺点，最佳实践 (10 min)
     b) 由 host 介绍自己的面试经过
- host goes through notebook (motivation, problem, data, model, 5 min)
- host 点评之前的评论 (5-10 min)
- host drive 自由讨论, facilitator 记笔记。(20-25 min )
- host / facilitator 进行小结 (10min)
- 确定下期活动的 host 和 facilitator (5min)

**Comment by suredream** [comment]
@stanghong  can you work more on my notes (plus yours) and turn it into a 1-2 page summary for today's meeting? Tks.

Jun 的开场白: 
- 深入的技术交流
- communication 练习 
- 社区
TH 的介绍, 48 hr take-home 
- insurance company
- doc requirements
	- clarification questions
	- interview processing (team leader position, 4-5 member)
	- expectation
		- hands-on
		- leadership exp
	- feedback, zoom meeting
			- Question for modeling, tech details (feature enginering)
			- fine-tuning
		- behavior
			- project management
			- multi-tasking
			- conflict, move project forward
			- mentoring

Q: 如何在做 take-home 之前跟公司明确具体的要求，有时找不到联系人？
some companies would cc  the hiring manager when sending you the take home exam. For those that don't, you can ask the person who you can contact if you have questions.

TH: 如何为 take-home 做准备？
1) 明确公司的具体需求，为 study 做准备
2) 熟悉一些 automation tools
3) 建立自己的 network，有一个自己的 review 组
逻辑上的硬伤，data 有没有 leaking，预处理（重要的 column 选择）
4) 时间分配上
	1) thinking fast 
	2) thinking slow 技术上的难点
5) 有用的 snippets

Feedback from Jun
- show the reasoning processing how to turn demand into a ML problem, select your metric carefully, defend your work
- discussion about distribution of the data, whether it fulfill the modeling questions, beyond the EDA
- potential overfiting of the training


Feedback from group:
- Xiangchen, Summary: "no show causes profit loss and resources wasted" -> “Department needs a solution to predict patience no show to deploy resources efficiently. I developed a solution by building a classification model".
- Xian: Put a little background in the notebook
- May: 顺序可以调整一下，那个5 minu summary deck， 在reference那里，我都没点开看。我觉得可以把这个改成，high level summary 放在notebook link 之前
- Abing: 请问tanghong在你面试过的公司中给home challenge的比例是多少？谢谢。
